basic/exercise5/script5.js

```diff
@@ -1,13 +1,16 @@
 'use strict';
 
 function findFirstYearStartingFromSunday(startYear, endYear) {
+    var years = [];
+        years.push(year);
```

Comment:
Not appropriate place for this instruction - you do not have year variable defined and you try to use iit.